src/lib/db/DatabaseManager.ts

```diff
@@ -78,29 +78,70 @@ class DatabaseManager {
 
   private async seedDatabase() {
     try {
-      await this.seed("categories", categories);
-      await this.seed("toolkit_items", toolkit);
-      await this.seed("needs_categories", needsCategories);
-      await this.seed("needs", needs);
-      await this.seed("next_actions", nextActions);
+      await this.oldSeedCategories();
+      // await this.seed("categories", categories);
+      await this.oldSeedToolkitItems();
+      // await this.seed("toolkit_items", toolkit);
+      // await this.seed("needs_categories", needsCategories);
+      // await this.seed("needs", needs);
+      // await this.seed("next_actions", nextActions);
+    } catch (error) {
+      console.error("Error during database seeding:", error);
+    }
+  }
```

Comment:
Change this to use generic seed methods that have been commented out and delete old seed methods